src/main/resources/fabric.mod.json

```diff
@@ -4,19 +4,16 @@
   "version": "${version}",
   "name": "StairAutoJump",
   "description": "Turns on auto jump when in/on stairs so that you can move faster!",
-  "authors": ["theRookieCoder", "RaptaG"],
```

Comment:
Why did you remove yourself?

Reply:
I didn't actually, I just made it clear that I'm not the author :)
https://github.com/theRookieCoder/StairAutoJump/pull/13/files#diff-882c6606823cf6c4e4114c1125850a04ed3180c48aef0e1467f1e4022b235db2R14

Reply:
Ahh I see